blog/content/post/2020-04-27-trenchboot-nlnet-release-04.md

```diff
@@ -0,0 +1,305 @@
+---
+title: 'TrenchBoot: Open Source DRTM.'
+abstract:
+cover: /covers/trenchboot-logo.png
+author: piotr.kleinschmidt
+layout: post
+published: false
+date: 2020-04-27
+archives: "2020"
+
+tags:
+  - trenchboot
+  - security
+  - open-source
+  - coreboot
+categories:
+  - Firmware
+  - Security
+
+---
+
+[Previous article](https://blog.3mdeb.com/2020/2020-04-03-trenchboot-nlnet-lz-validation/)
+showed how to enable DRTM on PC Engines apu2 platform and NixOS operating
+system. As project is constantly developed by us, there are regularly new
+features implemented. This article is showing how to update your DRTM and verify
+next project's requirements. Typically, new updates should be published every
+month as form of following blog posts and they will develop already configured
+system components. Therefore, **we strongly encourage to keep project up-to-date
+and keep track of TrenchBoot blog posts series**. If you haven't read previous
+articles, we recommend to catch up.
```

Comment:
@piotr-kleins some consistent recap of all blog posts from this series

Reply:
Added

Reply:
I've just added